styles.css

```diff
@@ -0,0 +1,662 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  transition: 0.3 linear;
+  font-family: 'Lato', sans-serif;
+}
+
+ul {
+  list-style: none;
+}
+
+:root {
+  --black: #272a31;
+  --red: #ec5242;
+  --gray: #d3d3d3;
+}
+
+html {
+  /* font-size: 62.5%; */
+  scroll-behavior: smooth;
+  overflow-x: hidden;
+  width: 100vw;
+}
+
+.desktop-menu {
+  display: none;
+}
+
+.mobile-menu {
+  /* display: none; */
+
+  /* color: #fff; */
+  position: absolute;
+}
+
+.nav--menu {
+  margin-top: 18px;
+  margin-left: 25px;
+}
+
+.nav--menu > div {
+  background-color: var(--black);
+  height: 4px;
+  width: 23px;
+  margin: 3px;
+}
+
+.menu {
+  font-size: 32px;
+  margin-top: 16px;
+  margin-left: 6%;
+}
+
+.m-menu {
+  /* display: none; */
+  background-color: var(--red);
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  overflow: hidden;
+  width: 100vw;
+  position: fixed;
+  z-index: 100;
+  top: 0;
+  left: -1000%;
+}
+
+.nav-links {
+  margin-top: 40px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.m-menu ul li {
+  margin: 15px 0;
+}
+
+.m-menu ul li a {
+  margin: 15px 0;
+  color: #fff;
+  font-weight: 700;
+  font-size: larger;
+}
+
+.nav-close {
+  color: #fff;
+  align-self: end;
+  margin-right: 15px;
+  margin-top: 10px;
+  font-size: 35px;
+}
+
+.show {
+  left: 0;
+
+  /* display: flex; */
+}
+
+.home {
+  height: auto;
+  width: 100vw;
+  background-image: url(./images/mobile-background.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: 90% 100%;
+  color: #ec5242;
+  padding: 50px 24px;
+}
+
+.page-title {
+  /* -webkit-background-clip: text; */
+  background-clip: text;
+  background-image: url(https://w0.peakpx.com/wallpaper/194/474/HD-wallpaper-red-and-orange-texture-red-aesthetic.jpg);
+  color: transparent;
+  background-position: right;
+
+  /* font-size: 2.5rem; */
+  font-weight: 950;
+  font-size: 30px;
+}
```

Comment:
- [ ] Great effort working on your capstone project 👍 , Currently, I notice your page title is only showing an image that is given to your page in form of a background image in order to achieve the transparency of the text. kindly uncomment the `-webkit-background-clip: text;` CSS property and the text should work just fine.

Your page 👇 

![image](https://user-images.githubusercontent.com/78634841/217746650-0edf45da-e957-4225-bed6-75797694c281.png)